src/RenderEngineManager.cc

```diff
@@ -375,7 +388,10 @@ bool RenderEngineManagerPrivate::LoadEnginePlugin(
   ignition::common::SystemPaths systemPaths;
 
   // Add default install folder.
+  systemPaths.SetPluginPathEnv(std::string(this->pluginPathEnv));
```

Comment:
I think this passes the string `IGN_GAZEBO_RENDER_ENGINE_PATH` but not the 'path' pointed to by the env variable.